docs/modules/ROOT/pages/interfaces.adoc

```diff
@@ -0,0 +1,177 @@
+= Interfaces and Dispatchers
+
+This section describes the interfaces OpenZeppelin Contracts for Cairo offer, and explains the design choices behind them.
+
+Interfaces can be found in the module tree under the `interface` submodule, such as `token::erc20::interface`. For example:
+
+```javascript
+use openzeppelin::token::erc20::interface::IERC20Dispatcher;
```

Comment:
Why `IERC20Dispatcher` instead of `IERC20`? The sentence is talking about the interface, not about dispatchers, right?